Feature/mkdocs with versioning (#333)
aeternity/aesophia: docs/index.md

# Introduction

> Sophia is a language in the ML family designed for smart contract development. It is strongly typed and has

Reviewer: ```suggestion
Sophia is a functional language designed for smart contract development. It is strongly typed and has
```

Reviewer: If you know ML (or another language in its family) you will recognize some syntax regardless, and if you don't, the "ML family" isn't going to provide you any insight 😅
